plugins/woocommerce-admin/client/homescreen/mobile-app-modal/style.scss

```
@@ -14,7 +14,7 @@
 	max-width: 964px;
 	min-width: 534px;
 	// Overrides the default modal max-height.
-	max-height: 663px;
+	max-height: 715px;
```

```suggestion
	max-height: unset;
```

Still getting a wee bit of scrollbar on step 2, I think this should resolve it